app/controllers/building_operators_controller.rb

```diff
@@ -0,0 +1,14 @@
+class BuildingOperatorsController < ApplicationController
+  def show
+    @building_operator = BuildingOperator.find(params[:id])
+    if building_operator_signed_in? && current_building_operator.id == @rmi_user.id
```

Comment:
what is @rmi_user, where is this defined? Could an rmi_user and building operator be signed in at the same time?